### DISCUSSION TITLE
Are there any plans for an android release?

### DISCUSSION THREAD

**Discussion by Ree11** [leolelego/brickie.swiftui#109] [General]
Just found out about your app and would love to have it. 
Definitely the best lego app I have seen so far and could not find any answer about this on the internet.

**Comment by leolelego** [comment]
Thanks @Ree11 I'm glad you like the app. Unfortunately this is side project for me to improve my iOS development skill and I don't develop for Android at all. 
But if you have skills or a friend on Android and he wants to do a similar app, I can help.
